pkg/agent/prommetrics.go

```diff
@@ -299,19 +300,34 @@ func makePerVMMetrics() (PerVMMetrics, *prometheus.Registry) {
 				"value",        // vmResourceValue: min, spec_use, status_use, max
 			},
 		)),
+		restartCount: util.RegisterMetric(reg, prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: "autoscaling_vm_restart_count",
+				Help: "Number of times that the VM has restarted",
+			},
+			[]string{
+				"vm_namespace", // .metadata.namespace
```

Comment:
I think this should be reused with other metrics. 

But can be left for the next time someone touches this code.

Reply:
I know we repeated label names for each metrics, although restart count's labels are slightly different. On the other hand, having the labels explicitly defined here is nice and straightforward. Anyway I'd rather not touch them on this PR and leave the potential improvements for the future.